tests/tiup-cluster/script/cmd_subtest.sh

```diff
@@ -67,6 +67,13 @@ function cmd_subtest() {
     tiup-cluster $client display "tmp-cluster-name"
     tiup-cluster $client rename "tmp-cluster-name" $name
 
+    # Test enable & disable
+    tiup-cluster $client exec $name -R tidb --command="systemctl status tidb-4000|grep 'enabled;'"
```

Comment:
I think the cluster name should also be added in the assertion.

Reply:
This action doesn't change cluster name